format/css.js

```diff
@@ -35,7 +35,7 @@ export default function css(tree, profile, options) {
 		outNode.afterOpen = options.format.between;
 		outNode.text = outNode.renderFields(value || null);
 
-		if (outNode.open) {
+		if (outNode.open && (!outNode.text || !outNode.text.endsWith(options.format.after))) {
```

Comment:
Since `format.after` depends on user preference and can be absent, the actual check should be `outNode.open && (!outNode.text || !outNode.text.endsWith(';'))`

Reply:
Done